roles/libvirt_configure/tasks/main.yml

```diff
@@ -63,9 +32,6 @@
 - name: 'copy syslinux files to /var/lib/tftpboot'
   shell: cp /usr/share/syslinux/{pxelinux.0,menu.c32,chain.c32} /var/lib/tftpboot
 
-- name: 'chown /var/lib/tftpboot for foreman-proxy user'
```

Comment:
Why remove this?

Reply:
Wasn't meaning to, I removed it temporarily to get it working on another machine. I'll replace it.